_data/simple-icons.json

```diff
@@ -8495,9 +8495,9 @@
         },
         {
             "title": "Ko-fi",
-            "hex": "FF5E5B",
-            "source": "https://more.ko-fi.com/brand-assets",
-            "guidelines": "https://more.ko-fi.com/brand-assets"
+            "hex": "FC5A17",
```

Comment:
```suggestion
            "hex": "FF5E5B",
```

Color is still `#FF5E5B` according to Ko-fi's branding page.

Reply:
It is #FF6433
![Screenshot 2024-10-08 at 21 15 39](https://github.com/user-attachments/assets/84f5d5af-a562-4f4e-b53d-15a130369538)

Reply:
https://ko-fi.com/post/October-Challenge-Rediscover-your-creative-joy-O4O2146NKM

Reply:
That isn't their branding guidelines though. It says this on the brand page:

![image](https://github.com/user-attachments/assets/8488a13e-1c0a-4466-a778-7cb52620213c)

Reply:
Ah,  my apologies ahah. Looks like they haven't even updated the logos there! Maybe we should reach out...